package.json

```diff
@@ -55,7 +57,7 @@
     "jest": "^24.3.1",
     "lint-staged": "^8.1.5",
     "prettier": "^1.16.4",
-    "rimraf": "^2.6.2"
+    "rimraf": "^2.7.1"
```

Comment:
How is it related?

Reply:
It's unrelated. I did not clean this change because it's a dev dependency and I think it's ok at here.

I guess that what you want is to make sure each PR only includes the related changes and keeps the changes minimum. I'll try to follow this rule in the future.

Please let me know if there are more rules that I should follow, thank you very much.

Reply:
> I guess that what you want is to make sure each PR only includes the related changes and keeps the changes minimum. I'll try to follow this rule in the future.

Yes please :slightly_smiling_face:

Reply:
> Please let me know if there are more rules that I should follow, thank you very much.

I'll tell you when necessary :slightly_smiling_face:

Reply:
Got it. 👌